Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM php:7.4-cli
```

Comment:
Tu mas php 7.4 ale v composer.json mas ^8.1 :)

Reply:
A dik robil som c&p. opravil som to lokalne mi docker build presiel